Sources/Sentry/include/HybridPublic/SentryDependencyContainer.h

```diff
@@ -90,7 +90,7 @@ SENTRY_NO_INIT
 @property (nonatomic, strong) SentryReachability *reachability;
 #endif // !TARGET_OS_WATCH
 
-- (SentryANRTracker *)getANRTracker:(NSTimeInterval)timeout;
+- (SentryANRTrackerV1 *)getANRTrackerV1:(NSTimeInterval)timeout;
 #if SENTRY_UIKIT_AVAILABLE
 - (SentryANRTrackerV2 *)getANRTrackerV2:(NSTimeInterval)timeout;
```

Comment:
l: We could make this functions also return the protocol.
So we can remove the `@class` reference from this file, and the `#include "SentryANRTrackerV1.h"` (and v2) from other files like `SentryANRTrackingIntegration.h`

Reply:
I can do this in the next PR. I'm still unsure if the dependency container should be the place to decide which version we use or if it should stay dump. I will cover this in the next PR.